roles/ansible-service-broker-setup/tasks/main.yml

```diff
@@ -0,0 +1,38 @@
+---
+
+- name: Get Ansible Broker project name
+  shell: oc get projects | grep -e ansible-service-broker -e automation-service-broker | awk '{print $1}'
+  register: broker_project_name
+
+- set_fact:
+    broker_project_name: '{{ broker_project_name.stdout }}'
```

Comment:
we probably want to check if the broker project actually exists. If it doesn't, we should print out a meaningful message to tell developers that the service broker is a prerequisite.